test/integration/recipe.test.js

```diff
@@ -59,7 +45,7 @@ describe('Recipes', function () {
         alloy: 'foobar3000',
         price: 'foobar3000',
         requiredCerts: [{ description: 'foobar3000' }],
-        supplier: 'foobar3000',
+        supplier: 'valid-1',
```

Comment:
looks like `newRecipe` repeats itself a fdew times, maybe this could be moved in terms of scope? (fixture)

Reply:
Yeah this was part of Dan's implementation. Agree it's not great